docs/network-policy-recommendation.md

```diff
@@ -0,0 +1,148 @@
+The goal of this feature is to aid the cluster admin or developers to secure
+their applications based on the idea of the Zero Trust model.
+
+## Deployment
+
+To use the NetworkPolicy recommendation feature, please follow this [doc](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md)
+to deploy Flow Exporter and Flow Aggregator first.
+Please update the `#podLabels: false` to `podLabels: true` inside
+[Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1),
+because policy recommendation feature needs Pod labels information to recommend network policies.
+
+Then, please follow these [deployment steps](network-flow-visibility.md#deployment-steps)
+to deploy the Grafana Flow Collector.
+
+After we finish the deployment of Grafana Flow Collector and can see some flow records in the Grafana UI,
+we can continue to deploy other necessary components of policy recommendation feature:
+
+We need to install the [Kubernetes Operator for Apache Spark](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator)
+in the cluster.
+Our policy recommendation logic is implemented as a [Spark](https://github.com/apache/spark) application,
+the Kubernetes Operator for Apache Spark helps us schedule the Spark job in the Kubernetes cluster.
+
+Please run the following command:
+
+```bash
+helm repo add spark-operator https://googlecloudplatform.github.io/spark-on-k8s-operator
+helm install policy-reco spark-operator/spark-operator --namespace flow-visibility --set image.tag=latest
+```
+
+This will install the Kubernetes Operator for Apache Spark into the `flow-visibility` namespace.
+
+Once we finish these deployment steps, we should see the `policy-reco-spark-operator` Pod running inside the `flow-visibility` namespace:
+
+```bash
+$ kubectl get pods -n flow-visibility
+NAME                                          READY   STATUS    RESTARTS   AGE
+policy-reco-spark-operator-56c4cb454c-4vhfh   1/1     Running   0          2m19s
+```
+
+## Perform NetworkPolicy Recommendation
+
+Currently, the NetworkPolicy recommendation feature only supports command line interaction through theia.
+Theia is the command-line tool which provides access to Theia network flow visibility capabilities.
+To get more information about theia, please refer to this [doc](./theia.md).
+
+We have 3 theia commands for the policy recommendation feature:
+
+- `theia policy-recommendation run`
+- `theia policy-recommendation status`
+- `theia policy-recommendation retrieve`
+
+Or you could use `pr` as a short alias of `policy-recommendation`:
+
+- `theia pr run`
+- `theia pr status`
+- `theia pr retrieve`
+
+To see all options and usage examples of these commands, you may run `theia policy-recommendation [subcommand] --help`. In the following sections, we will go through a simple example to utilize these commands with default options.
```

Comment:
Nit: Perhaps the last sentence in not needed.

Reply:
Thanks, removed.